docs/WindowsBuild.md

```diff
@@ -1,408 +1,78 @@
-# Building Swift on Windows
+# Building the Swift toolchain on Windows
 
-Visual Studio 2017 or newer is needed to build Swift on Windows. The free Community edition is sufficient to build Swift.
+The following is a step by step guide on how to build the Swift toolchain on x86/x64/ARM64 Windows.
 
-The commands below (with the exception of installing Visual Studio) must be entered in the "**x64 Native** Tools Command Prompt for VS2017" (or VS2019, VS2019 Preview depending on the Visual Studio that you are using) in the Start Menu. This sets environment variables to select the correct target platform.
+## Installing the dependencies
 
-## Install dependencies
+### Visual Studio Build Tools
 
-### Visual Studio
+Visual Studio Build Tools 2017 or newer is needed to build Swift on Windows. The free Community edition is sufficient to build the Swift toolchain. We will use the 2022 Community Edition.
 
-An easy way to get most of the tools to build Swift is using the [Visual Studio installer](https://www.visualstudio.com/downloads/). This command installs all needed Visual Studio components as well as Python, Git, CMake and Ninja:
+In a PowerShell shell, run the following command to install the required build tools.
 
+```powershell
+curl.exe -sOL https://aka.ms/vs/17/release/vs_BuildTools.exe
+./vs_BuildTools.exe `
```

Comment:
Is there a reason that you switched to the Build Tools? For development work, I think that we should stick with the full IDE for debugging support (LLDB is not usable today).